engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Vector.java

```diff
@@ -67,6 +63,26 @@ public final Object toArray() {
     return this.storage;
   }
 
+  @Builtin.Method(name = "slice", description = "Returns a slice of this Vector.")
+  @Builtin.Specialize
+  @Builtin.WrapException(from = UnsupportedMessageException.class, to = PanicException.class)
+  public final Vector slice(long start, long end, InteropLibrary interop)
```

Comment:
Is the `InteropLibrary` cached or uncached?

Reply:
If you look at the generated node it is cached